File: algorithms/warmup/python/simple-array-sum.py
#!/bin/python

import sys

def simpleArraySum(n, ar):
    return sum(ar[:n])
    # Complete this function

n = int(raw_input().strip())
ar = map(int, raw_input().strip().split(' '))
result = simpleArraySum(n, ar)
print(result)

File: algorithms/warmup/ruby/simple-array-sum.rb
#!/bin/ruby

def simpleArraySum(n, ar)
    # Complete this function
    sum = 0
    (0...n).each { |i| sum += ar[i] }
    return sum
end

n = gets.strip.to_i
ar = gets.strip
ar = ar.split(' ').map(&:to_i)
result = simpleArraySum(n, ar)
puts result;
